src/FixedDataTableTranslateDOMPosition.js

```diff
@@ -20,9 +22,18 @@ function FixedDataTableTranslateDOMPosition(/*object*/ style, /*number*/ x, /*nu
     style.left = x + 'px';
     style.top = y + 'px';
   } else {
+    if (BrowserSupportCore.hasCSSTransforms() && Locale.isRTL()) {
```

Comment:
what's the use of `BrowserSupportCore.hasCSSTransforms` ?

Reply:
Inside `translateDOMPositionXY.js`, when `BrowserSupportCore.hasCSSTransforms` is false, the result of the translate is an inline style change of `left` and `top`, whereas if that condition is true, a `translate3d` is performed.

Basically, it created this situation where the `x` value _should_ be flipped when `BrowserSupportCore.hasCSSTransforms` is true, but should not be when it's false (since the value should remain unchanged and be moved from `style.left` to `style.right`)